[2023-08-30T00:46:48.189+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T00:46:48.310+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table scheduled__2023-08-29T00:00:00+00:00 [queued]>
[2023-08-30T00:46:48.312+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-08-30T00:46:48.312+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-08-30T00:46:48.313+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-08-30T00:46:48.374+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-08-29 00:00:00+00:00
[2023-08-30T00:46:48.564+0000] {standard_task_runner.py:55} INFO - Started process 43174 to run task
[2023-08-30T00:46:48.593+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'scheduled__2023-08-29T00:00:00+00:00', '--job-id', '13', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmpx4h569sm']
[2023-08-30T00:46:48.616+0000] {standard_task_runner.py:83} INFO - Job 13: Subtask create_table
[2023-08-30T00:46:49.061+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table scheduled__2023-08-29T00:00:00+00:00 [running]> on host f65dd69de24d
[2023-08-30T00:46:49.351+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-08-29T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-08-29T00:00:00+00:00
[2023-08-30T00:46:49.385+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-08-30T00:46:49.405+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users (
                firstname TEXT NOT NULL,
                lastname TEXT NOT NULL,
                country TEXT NOT NULL,
                username TEXT NOT NULL,
                password TEXT NOT NULL,
                email TEXT NOT NULL
            );
        , parameters: None
[2023-08-30T00:46:49.411+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-08-30T00:46:49.432+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230829T000000, start_date=20230830T004648, end_date=20230830T004649
[2023-08-30T00:46:49.614+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-08-30T00:46:49.895+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
